tasks-by-users/Rehan-Khan-Engineer/08-Custom select box/style.css

```diff
@@ -0,0 +1,85 @@
+body {
+  margin: 0;
+  padding: 0;
+
+  box-sizing: border-box;
+
+  background-color: #004882;
+}
+
+.box {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.box select {
+  background-color: #0563af;
+  color: #fff;
+
+  height: 3rem;
+  width: 15rem;
+  padding: 10px;
+
+  font-size: 16px;
+
+  outline: none;
+  border: none;
+
+  /* Important */
+  -webkit-appearance: button;
```

Comment:
this property will only work in webkit browsers.

Reply:
So what should i do @shaikh-7abish

Reply:
make this property usable for other browsers... @Rehan-Khan-Engineer